test/e2e-ocl/onclusterlayering_test.go

```diff
@@ -319,65 +318,71 @@ func TestMachineConfigPoolChangeRestartsBuild(t *testing.T) {
 // MachineOSConfig with the expectation that the failed build and its objects
 // will be deleted and a new build will start in its place.
 func TestGracefulBuildFailureRecovery(t *testing.T) {
-	ctx, cancel := context.WithCancel(context.Background())
-	t.Cleanup(cancel)
+	/*
+		QOCL: Need a new failure mode for this as BaseImagePullspec field doesn't exist anymore
```

Comment:
With the removal of `BaseImagePullSpec`, I'm not sure the test can be initiated in the same way here, so I'm open to ideas here

Reply:
We can do a Containerfile with invalid syntax instead. It will just take a bit longer to produce the failure.